standalone-integration/src/test/java/com/ericsson/bss/cassandra/ecchronos/standalone/TestBase.java

```diff
@@ -82,9 +94,15 @@ public static void initialize() throws IOException
     @AfterClass
     public static void cleanup() throws IOException
     {
-        myJmxConnectionProvider.close();
-        myAdminNativeConnectionProvider.close();
-        myNativeConnectionProvider.close();
+        if (myJmxConnectionProvider != null) {
```

Comment:
Code checks want curly bracket on its own line, not trailing statements.

(same below)